kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -209,6 +212,13 @@
       "If true, no fields in any produced BigQuery schema will be REQUIRED. All "
       + "non-nullable avro fields will be translated as NULLABLE (or REPEATED, if arrays).";
 
+  public static final String TABLE_CREATE_CONFIG =                     "autoCreateTables";
+  private static final ConfigDef.Type TABLE_CREATE_TYPE =              ConfigDef.Type.BOOLEAN;
+  public static final boolean TABLE_CREATE_DEFAULT =                   false;
```

Comment:
Can you default this to true? I feel that it's a more user-friendly default.

Reply:
Make sense! Updated accordingly~